xalia/Win32/Win32Connection.cs

```diff
@@ -109,7 +109,7 @@ public string GetElementName(IntPtr hwnd, int idObject=OBJID_CLIENT, int idChild
                 case OBJID_CLIENT:
                     if (idChild == CHILDID_SELF)
                         goto case OBJID_WINDOW;
-                    return $"hwnd-{hwnd.ToInt64():x}-{idChild}";
+                    return $"acc2-{hwnd.ToInt64():x}-{idChild}";

```

Comment:
It feels a little bit odd to me to have this be hardcoded as "acc2", I guess the assumption is that the only thing that would be passed into `GetElementName` here is an IAccessible2. Might be worth adding an argument to the function to set this, but feel free to ignore this. :)

Reply:
I guess instead of that, we could do away with "acc2" entirely and use hwnd-HWND-CHILDID

Reply:
At the moment, IAccessible2 is the only way we get a childid that we know is stable (so other cases will use a different naming scheme), but if that changes then "acc2" will be inaccurate.

Reply:
Removed that diff, so "hwnd-" will be used in this case now.